config/betterquesting/DefaultQuests/Quests/Tier05Steam-AAAAAAAAAAAAAAAAAAAAAg==/ToolForge-AAAAAAAAAAAAAAAAAAAC5Q==.json

```diff
@@ -36,41 +36,6 @@
   },
   "tasks:9": {
     "0:10": {
-      "partialMatch:1": 1,
-      "autoConsume:1": 0,
-      "groupDetect:1": 0,
-      "ignoreNBT:1": 0,
-      "index:3": 0,
-      "consume:1": 0,
```

Comment:
I believe this will error since the index:3 of the tasks has the wrong number and will go out of bounds. Since you delete the first one, but don’t renumber the ones in the other tasks.

Reply:
Should hopefully be fixed my removing and re-adding them.